style.css

```diff
@@ -2324,38 +2324,29 @@ object {
 	text-align: center;
 	vertical-align: top;
 	width: 100%;
+	margin: 0 0 1.5em 0;
+	padding: 0 .5em;
+	width:50%;
 }
 
-.gallery-columns-2 .gallery-item {
-	max-width: 50%;
-}
-
-.gallery-columns-3 .gallery-item {
-	max-width: 33.33%;
-}
-
-.gallery-columns-4 .gallery-item {
-	max-width: 25%;
-}
-
-.gallery-columns-5 .gallery-item {
-	max-width: 20%;
+.gallery-item a {
+	display:inline-block;
+	border-bottom:none;
+	background:none;
+	border:10px solid transparent;
 }
```

Comment:
Properties should be followed by a colon and a space e.g. `background: none`, not `background:none`